.travis.yml

```diff
@@ -42,7 +42,7 @@ deploy:
   on:
     branch: master
     condition:
-      - "( ${TRAVIS_OS_NAME}--${CC} = 'linux--gcc-4.8' ) || ( ${TRAVIS_OS_NAME}--${CC} = 'osx--clang' ) && echo \"published nupic_core-${TRAVIS_COMMIT}-${PLATFORM}64.tar.gz\" "
```

Comment:
You can't have an `echo` within this condition because travis expects it to be a boolean expression.